libs/design/switch/src/switch/switch.component.ts

```diff
@@ -7,14 +9,80 @@ import {
   Output,
 } from '@angular/core';
 
+import { DAFF_LOADING_ICON_COMPONENTS } from '@daffodil/design/loading-icon';
+import { DaffToastService } from '@daffodil/design/toast';
+
 @Component({
   selector: 'daff-switch',
   changeDetection: ChangeDetectionStrategy.OnPush,
   templateUrl: './switch.component.html',
   styleUrls: ['./switch.component.scss'],
   standalone: true,
+  imports: [
+    CommonModule,
```

Comment:
Don't import `CommonModule` in standalone components. Only import what's necessary.